tests/short_lived_metrics_gen.py

```diff
@@ -0,0 +1,93 @@
+import numpy as np
+import csv
+import time
+import random
+import socket
+import argparse
+import string
+
+parser=argparse.ArgumentParser()
+
+parser.add_argument('--host', default='localhost')
+parser.add_argument('--port', default=2003, type=int)
+parser.add_argument('--datapoints', default=1440*3, type=int)
+parser.add_argument('--paths-outer', default=100, type=int)
+parser.add_argument('--paths-inner', default=100, type=int)
+parser.add_argument('--total-metrics', default=100, type=int)
+parser.add_argument('--total-pods', default=100, type=int)
```

Comment:
I would rename this to make it more generic. Same about other variations. Maybe something like *first*, *second*. *third*, *fourth*? Different people may want to use this differently.